drivers/tee/optee/rpmb.c

```diff
@@ -106,6 +108,10 @@ static u32 rpmb_get_dev_info(u16 dev_id, struct rpmb_dev_info *info)
 		return TEE_ERROR_GENERIC;
 
 	memcpy(info->cid, mmc->cid, sizeof(info->cid));
+
+	for (i = 0; i < 4; i++)
```

Comment:
I feel like the `4` here should be the array length of info->cid (# of u32 elements)

Reply:
yep

Reply:
I'll write it differently and post it upstream